packages/view-types/description/package.json

```diff
@@ -32,7 +32,8 @@
   "dependencies": {
     "@material-ui/core": "catalog:",
     "@vitessce/constants-internal": "workspace:*",
-    "@vitessce/vit-s": "workspace:*"
+    "@vitessce/vit-s": "workspace:*",
+    "react-markdown": "^9.0.1"
```

Comment:
Would you be able run `pnpm install` from the root of the repository and then commit the modified `pnpm.lock` file?